src/Webserver/DefaultConfiguration.php

```diff
@@ -34,4 +34,9 @@ public function getDocroot()
     {
         return $this->configuration->getDocroot() ?: $this->basePath . Configuration::DEFAULT_DOCROOT;
     }
+
+    public function getRouter()
+    {
+        return $this->configuration->getRouter() ?: Configuration::DEFAULT_ROUTER;
```

Comment:
I don't think this is the correct behaviour.

When the web server is not given a router, it will use `index.html` *or* `index.php`. In these cases, `index.php` is not a router in the same sense as when you provide a file at the CLI, and we'll get unexpected behaviour when the front page of the site is `index.html`.

This should default to Null and you should not provide the extra argument if one is not specified in the config.